### DISCUSSION TITLE
Color Picker missing "CIELAB" color space, also referred to as " L*a*b* "

### DISCUSSION THREAD

**Discussion by donatopace** [d2phap/ImageGlass#1542] [General] [answered]
Hello,
first of all congratulations for the great work you are doing!

Can you add
"CIELAB" color space, also referred to as " L*a*b* "

it is very important for those who work with images and color management.
"Lab" expresses color as three values:
L* for perceptual lightness
and a* and b* for the four [unique colors](https://en.wikipedia.org/wiki/Unique_hues) of human vision: red, green, blue and yellow. CIELAB was intended as a [perceptually uniform](https://en.wikipedia.org/wiki/Perceptually_uniform) space, where a given numerical change corresponds to a similar perceived change in color.

Thanks.

**Comment by d2phap** [comment]
Just created #1543 as a feature request

**Reply by d2phap** [reply, marked as answer]
Implemented in [ImageGlass 9.0 beta 5](https://github.com/d2phap/ImageGlass/releases/tag/9.0.5.726)

![image](https://github.com/d2phap/ImageGlass/assets/3154213/87a82011-22c3-490e-b3f6-e2a3b97ddcca)
